expression/aggregation.go

```diff
@@ -76,11 +85,23 @@ func NewAggFunction(funcType string, funcArgs []Expression, distinct bool) Aggre
 
 type aggCtxMapper map[string]*ast.AggEvaluateContext
 
+// AggFunctionMode stands for the aggregation function's mode.
+type AggFunctionMode int
+
+const (
+	// CompleteMode function accepts origin data.
+	CompleteMode AggFunctionMode = iota
```

Comment:
`CompleteMode` is a little misleading, looks like a mode that is special.
How about rename it to `OriginMode` according to the comment?